ci/ray_ci/automation/BUILD.bazel

```diff
@@ -212,3 +222,17 @@ py_test(
         ":automation",
     ],
 )
+
+py_test(
+    name = "test_determine_microcheck_step_ids",
+    srcs = ["test_determine_microcheck_step_ids.py"],
+    exec_compatible_with = ["//:hermetic_python"],
+    tags = [
+        "ci_unit",
+        "team:ci",
+    ],
+    deps = [
+        ci_require("pytest"),
+        ":automation",
```

Comment:
I think you need to declare `determine_microcheck_step_ids` as its dependency in some way. otherwise, it won't be able to import it.